src/components/visualizations/table/Table.tsx

```diff
@@ -1,32 +1,32 @@
 // (C) 2007-2018 GoodData Corporation
 import * as React from 'react';
-import * as Measure from 'react-measure';
+import Measure from 'react-measure';
 import { IHeaderPredicate } from '../../../interfaces/HeaderPredicate';
 
 import { TableVisualization, ITableVisualizationProps } from './TableVisualization';
 
-export interface IDimensions {
-    height: number;
-    width: number;
-}
-
 export interface ITableProps extends ITableVisualizationProps {
     containerWidth?: number;
     containerHeight?: number;
     drillablePredicates?: IHeaderPredicate[];
 }
 
+
 export class Table extends React.Component<ITableProps> {
     public render(): JSX.Element {
         const { containerHeight, containerWidth } = this.props;
         return (
-            <Measure>
-                {(dimensions: IDimensions) => (
-                    <div className="viz-table-wrap" style={{ height: '100%', width: '100%', position: 'relative' }}>
+            <Measure client={true}>
+                {({ measureRef, contentRect }: any) => (
+                    <div
+                        className="viz-table-wrap"
+                        style={{ height: '100%', width: '100%', position: 'relative' }}
+                        ref={measureRef}
+                    >
                         <TableVisualization
                             {...this.props}
-                            containerHeight={containerHeight || dimensions.height}
-                            containerWidth={containerWidth || dimensions.width}
+                            containerHeight={containerHeight || contentRect.client.height}
+                            containerWidth={containerWidth || contentRect.client.height}
```

Comment:
There is a typo, it should be: `containerWidth={containerWidth || contentRect.client.width}`